lib/manageiq/api/common/rbac/access.rb

```diff
@@ -0,0 +1,68 @@
+module ManageIQ
+  module API
+    module Common
+      module RBAC
+        class Access
+          attr_reader :acl
+          DEFAULT_LIMIT = 500
+          def initialize(resource, verb)
+            @resource = resource
+            @verb     = verb
+            @regexp   = Regexp.new(":(#{Regexp.escape(@resource)}|\\*):(#{Regexp.escape(@verb)}|\\*)")
+            @app_name = ENV["APP_NAME"]
+          end
+
+          def process
+            Service.call(RBACApiClient::AccessApi) do |api|
+              @acl = Service.paginate(api, :get_principal_access, {:limit => DEFAULT_LIMIT}, @app_name).select do |item|
+                Rails.logger.info("Found ACL: #{item}")
```

Comment:
This will get written out for every ACL.  Is that intentional?  a) that seems like a lot of logging and b) it was also print for the ACLs that don't match.

Reply:
Removed